styles/styles.css

```diff
@@ -0,0 +1,239 @@
+/* variables and imports */
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Oleo+Script+Swash+Caps:wght@400;700&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+:root {
+    --color-purple: #5722CD;
+    --color-blue: #4845E4;
+    --color-pink: #ED4385;
+    --textcolor-standard-black: #0C0C0C;
+    --textcolor-special-lightpurple: #F2F0FF;
+    --bg-color-article-lightgrey: #F3F6F9;
+    --textcolor-article-grey: #7A7A7A;
+}
+
+/*Global Styles*/
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body {
+    font-size: 12px;
+    font-family: "Roboto", serif;
+    font-weight: 400;
+    font-style: normal;
+}
+h1, h2 {
+    font-family: "Merriweather", serif;
+    font-weight: 700;
+    font-style: normal;
+}
+a {
+    text-decoration: none;
+    color: var(--color-purple);
+}
+header {
+    background-color: var(--color-blue );
+    color: white;
+}
+nav {
+    height: 5rem;
+    padding: 10px 0 10px 0;
+}
+ul {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width:35rem;
+    column-gap: 0.5rem  ;
+}
+li {
+    text-align: center;
+    font-size : 1.5rem;
+}
+h1 {
+    font-size: 3rem;
+}
+form {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: space-evenly;
+    align-items: center;
```

Comment:
Met `align-items: flex-start` had je de labels links kunnen plaatsen zoals in het voorbeeld.